backend/app/Http/Controllers/CheckoutController.php

```diff
@@ -27,9 +27,7 @@ public function store(StoreCheckoutRequest $request)
 
     public function show(Checkout $checkout)
     {
-        return response()->json([
-            'data' => new CheckoutResource($checkout->load('paymentType'))
-        ]);
+        new CheckoutResource($checkout->load(['paymentType', 'shipper_id', 'checkoutDetails.order']));
```

Comment:
Parang walang return?

Reply:
Oh, sorry I missed it. I'll resolve it right away